examples/resources/elasticstack_elasticsearch_security_system_user/resource.tf

```
@@ -0,0 +1,16 @@
+provider "elasticstack" {
+  elasticsearch {}
+}
+
+resource "elasticstack_elasticsearch_security_system_user" "kibana_system" {
+  username = "kibana_system"
+
+  // use hashed password: see https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-user.html#security-api-put-user-request-body
```

I initially read this comment as if I couldn't set the plaintext password. 
```suggestion
  // For details on how to generate the hashed password see https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-put-user.html#security-api-put-user-request-body
```